plugins/org.jboss.tools.openshift.client/.classpath

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry exported="true" kind="lib" path="lib/openshift-restclient-java-5.9.0.Final.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/openshift-restclient-java-5.9.1-SNAPSHOT.jar"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
```

Comment:
we now have 5.9.0.Final available, we should preferably switch to the final artifact in this (or an additional PR) for the upcoming milestone.

Reply:
Yes but I have a pending PR so it will land in 5.9.1.Final

Reply:
+1

Reply:
so we'll have to release 5.9.2.Final later today.